pyvista/plotting/themes.py

```diff
@@ -18,7 +18,7 @@
 >>> loaded_theme = pv.load_theme('my_theme.json')  # doctest:+SKIP
 
 Create a custom theme from the default theme and load it into
-pv.
+pyvista.
```

Comment:
Good that this is reverted, I hope it didn't mess up much other examples or doc strings?

Reply:
Sorry for bothering you. I believe that this revert can be done in the following steps to avoid any omissions.

1. replace all `pv.` with `pyvista`.
1. replace `pv.` with `pyvista.` again only where errors occur in the CI and doctest checks.

As for 2, it is a bit pain in the ass, but it is sure to have CI check it. It has been verified with the previous change that `import pyvista as pv` is defined in all places. Please wait as I will do all of them.